docs/organizations/verify-user-permissions.mdx

```diff
@@ -6,26 +6,16 @@ description: A collection of utility functions and components in order to allow
 # Verify the active user's permissions in an organization
 
 <Callout type="info">
-  The following authorization checks are predicated on a user having an active organization. Without this, they will likely always evaluate to false by default. Clerk's built-in [`<OrganizationSwitcher/>`](/docs/components/organization/organization-switcher) component and `[useOrganizationList()](/docs/references/react/use-organization-list).setActive({ organization: <orgId> })` method are two available options to set an organization as active for a user.
+  The following authorization checks are predicated on a user having an active organization. Without this, they will likely always evaluate to false by default. Clerk's built-in [`<OrganizationSwitcher/>`](/docs/components/organization/organization-switcher) component and[`useOrganizationList().setActive({ organization: <orgId> })`](/docs/references/react/use-organization-list) method are two available options to set an organization as active for a user.
 </Callout>
 
 Clerk provides two kinds of helpers for authorization:
 
-- **`protect` helpers** are opinionated and will block additional processing for unauthorized users, either by not rendering children or by throwing errors. The [`<Protect>` component](/docs/components/protect) and the experimental `auth().protect()` method fall into this category.
-- **`has` helpers** are more composable and flexible. They allow you to check and see what permissions a user _has_. You can decide what to do if the user lacks the permission.
+- `protect()` helpers are opinionated and will block additional processing for unauthorized users, either by not rendering children or by throwing errors. The [`<Protect>` component](/docs/components/protect) and the experimental [`auth().protect()`](/docs/references/nextjs/auth#protect) method fall into this category.
```

Comment:
```suggestion
1. **Immediately blocking unauthorized users**.
  - Use the [`<Protect>`](/docs/components/protect) component to prevent content from rendering if the active user is unauthorized.
  - Call [`auth().protect()`](/docs/references/nextjs/auth#protect) to throw an error if the active user is unauthorized.
```